### PR TITLE
Fix GitHub Actions - Clean Lint workflow- bug - if gpg keys not found

### PR DESCRIPTION
**Pull request by anirudTT**
### This PR addresses the GitHub Actions - Clean Lint workflow:
Ensuring it no longer errors out if a GPG key is not found. It will proceed with all the steps and automatically commit changes without requiring a verified GPG signature. This adjustment is needed for scenarios where a PR is opened from an external fork. Previously, GitHub Actions would report a failure because it runs on the forked repository, where it definitely fail to locate the GPG key and passphrase; since it will not be present there.

### CODE REVIEW

**Review by anirudTT — COMMENTED** [review]
moving to draft to make additional corrections

File: .github/workflows/clean-style-lint.yml [hunk 1]
```diff
@@ -22,20 +27,37 @@ jobs:
           make clean
           make clean_tt
           make style
-      - name: Import GPG key
+      
+      #!TODO: remove in future -- after a few runs
```
Comment (milank94) [inline .github/workflows/clean-style-lint.yml:31]: Do you need this block of commented out code? What's the intent of keeping it in?

Comment (anirudTT) [reply .github/workflows/clean-style-lint.yml]: I don't need it ; forgot to remove it :)

**Review by anirudTT — COMMENTED** [review]
[remove commented block of code](https://github.com/tenstorrent/tt-buda-demos/pull/45/commits/96576a2b6e2a8c9b70fbfe242bcb1fdfbccb910b)

**Review by milank94 — APPROVED** [review]
LGTM

**Pull request merged**